CONTRIBUTING.md

```diff
@@ -112,6 +112,14 @@ Bugs and tasks are obviously the easiest to get started with, but don't feel afr
 
 Generally, we like to assign issues that generally belong to somebody's area of expertise to the team member that owns that area. This doesn't mean the community can't jump in -- they should reach out and have a chat with the assignee to see if it'd okay to take. If an issue was assigned more than a month ago, there's a good chance it's fair game to try yourself.
 
+### Contributing to Windows Customization
+
+If you'd like to suggest a new feature/enhancement in Windows Customization, **you must first file an issue with the provided `Windows Customization` template**. This will help us understand what you're looking for and why, and will help us ensure that the feature is something that we can support in the long run. For changes that rely on registry key behaviors that are undocumented, we may first have to review with internal stakeholders how to support the desired functionality and may not be able to support them in the long run. In these cases, we may have to modify your PR with a different approach after chatting with the internal teams.
```

Comment:
Suggestion: link the reference to the template to the "open issue" link
`https://github.com/microsoft/devhome/issues/new?template=feature_request_windows_customization.yml`